app/services/pafs_core/spreadsheet_service.rb

```diff
@@ -131,8 +131,8 @@ def add_project_to_sheet(sheet, project, row_no)
             value = use_value ? project.send(col[:field_name]) : 0
             sheet[row_no][column_index(col[:column])].change_contents(value)
           rescue StandardError
-            raise "Boom - Project (#{project.slug}) Row no (#{row_no}) col (#{col})" \
-                  " field_name (#{col[:field_name]}) value (#{value})"
+            raise "Boom - Project (#{project.slug}) Row no (#{row_no}) col (#{col}) " \
```

Comment:
That's interesting, there is a new bug for spreadsheet exports https://eaflood.atlassian.net/browse/RUBY-2166